config/initializers/menus.rb

```diff
@@ -42,7 +42,7 @@
             last: true
   menu.push :help, OpenProject::Info.help_url,
             last: true,
-            caption: I18n.t('label_help'),
+            caption: '',
```

Comment:
Accepting a `title` parameter that defaults to the `caption` sounds pretty clean to me (and should be straight forward using named parameters... probably it is old code and named parameters won't work...).

Using the `html` hash sounds like a reasonable workaround, but TBH I can't tell if this is rather a neat trick or a dirty hack.